modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/search/insights/view.jsp

```diff
@@ -72,11 +80,39 @@ SearchInsightsDisplayContext searchInsightsDisplayContext = (SearchInsightsDispl
 					persistState="<%= true %>"
 					title="response-string"
 				>
-					<code>
-						<%= HtmlUtil.escape(searchInsightsDisplayContext.getResponseString()) %>
-					</code>
+					<clay:button
+						displayType="secondary"
+						icon="copy"
+						label="copy-to-clipboard"
+						onClick='<%= liferayPortletResponse.getNamespace() + "copyToClipboard('insightsResponse');" %>'
```

Comment:
I think this would look cleaner if you used `<portlet:namespace />` instead. (See my comment below why I added `<portlet:namespace />` to the parameter)

```suggestion
						onClick="<portlet:namespace />copyToClipboard('<portlet:namespace />insightsResponse');" %>"
```

Reply:
Oh it looks like I'm getting a syntax error using `onClick="<portlet:namespace />copyToClipboard('<portlet:namespace />insightsResponse');"` , and no action when using `onClick="<portlet:namespace />copyToClipboard('insightsResponse');"`  maybe it doesn't work with `<portlet:namespace />` ? I'm not sure when I can substitute one for the other.

Maybe we can go with a variable for `liferayPortletResponse.getNamespace() + insightsResponse`, that might make it easier to read?